plugins/git-build.js

```diff
@@ -35,9 +35,18 @@ function buildConsumer(config, cimpler, repoPath) {
             BUILD_STATUS: build.status
          },
          timeout: config.timeout || 0,
-         maxBuffer: config.maxBuffer || 1024 * 1024 * 2
+         maxBuffer: config.maxBuffer || 1024 * 1024 * 2,
+         shell: '/bin/bash'
       },
-      killChildrenOnExit = "trap '[ $(jobs -p) ] && kill $(jobs -p)' EXIT",
+      killChildrenOnExit = `
+         trap '
+            [ -z "$(jobs -p)" ] || kill -SIGTERM $(jobs -p)
+            && sleep 2
+            && [ -z "$(jobs -p)" ] || kill -SIGKILL $(jobs -p)
+            ; wait $(jobs -p)
+         '
+         EXIT`
```

Comment:
I don't think backticks are a good choice here. You're one bash `${VAR:-default}` away from blowing up on template literal expansion. It's not a bug or a blocker _now_, but it could be a nasty surprise later.